pkg/sql/executor_normal_stmt.go

```diff
@@ -31,17 +31,16 @@ func runNormalStmt(wr *pipe.Writer, slct string, db *database.DB) error {
 
 // TODO(weiguo): isQuery is a hacky way to decide which API to call:
 // https://golang.org/pkg/database/sql/#DB.Exec .
-// We will need to extend our parser to be a full SQL parser in the future.
-func isQuery(slct string) bool {
-	s := strings.ToUpper(strings.TrimSpace(slct))
-	has := strings.Contains
-	if strings.HasPrefix(s, "SELECT") && !has(s, "INTO") {
+// The implementation is based on
+// https://github.com/xo/usql/blob/490ef67b5accd38d53cbe0a333bef85480a0a716/drivers/qtype.go#L8-L22
+// We may need to call SQLFlow parser to check the statement type in the future.
+func isQuery(stmt string) bool {
```

Comment:
@shendiaomo `isQuery` is a hack according to the function comment. It means that we need to get rid of it ASAP. It is not a good idea to implement any new idea by extending it.

Suppose that the user writes a SQL statement like

```sql
SELECT * FROM sqlflow_history WHERE stmt="SELECT * FROM a TO TRAIN b INTO c";
```

the following condition would easily get confused:

```go
if strings.HasPrefix(s, "SELECT") && !strings.Contains(s, "INTO") {
```

Reply:
I'll check it out.